community/kernel/src/main/java/org/neo4j/kernel/impl/coreapi/schema/SchemaImpl.java

```diff
@@ -132,7 +133,8 @@ private IndexDefinition descriptorToDefinition( final ReadOperations statement,
     {
         try
         {
-            Label label = label( statement.labelGetName( index.schema().getLabelId() ) );
+            //TODO we use a zero index here to avoid bleeding into core api. We need to figure out how to deal with the core api in regards to fulltext indexes.
```

Comment:
TODO again

Reply:
This is the bleeding edge towards the not-yet-supported core api.